Resolve syncing conflicts from repo_sync_working_branch to live (#2300)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/provision-sss-lookbook.md

---
title: Provision the SharePoint Success Site from the look book
ms.reviewer: 
ms.author: hokavian
author: Holland-ODSP
manager: pamgreen
audience: Admin
f1.keywords:
- NOCSH
ms.topic: article
ms.service: sharepoint-online
localization_priority: Priority
ms.collection:  
- Strat_SP_modern
- M365-collaboration
search.appverid:
- SPO160
- MET150
ROBOTS: NOINDEX, NOFOLLOW
description: "Provision the SharePoint Success Site from the look book"
---

# Provision the SharePoint Success Site from the look book

> Once you’ve confirm the person provisioning and the tenant being provsioned meet the following prerequisites, you are ready to provision:

Reviewer: ```suggestion
Once you’ve confirmed the person provisioning and the tenant being provisioned meet the following prerequisites, you are ready to provision:
```

- You are using *at minimum* SharePoint tenant administrator credentials
- Your tenant has the App Catalog installed
- You are an administrator of the App Catalog
- Your tenant has version 4.0 of the M365 learning pathways provisioned

## Start provisioning from the look book

> If you already have Learning Pathways 4.0 or greater provisioned on your tenant, you can provision Learning Pathways from the SharePoint Look book site. 

Reviewer: ```suggestion
If you already have Learning Pathways 4.0 or greater provisioned on your tenant, you can provision Learning Pathways from the SharePoint look book site. 
```

1. Go to the [look book](https://provisioning-test.sharepointpnp.com/details/0b860749-56a0-4c4c-992c-536d56d9accf) and select **Add to your tenant**

![Image of the SharePoint Success Site look book page](media/sss-lookbook-add.png)

2. Fill out the email and URL address sections and select **Provision**

![Image of the SharePoint Success Site look book page details](media/sss-lookbook-details.png)

3.	Select **Confirm**
4.	When provisioning is complete, you’ll see the following message:

![Image of the provisioning confirmation message](media/sss-sss-complete.png)

5.	Next, navigate to your tenant's Learning Pathways site
6.	Select **Home > Learning Pathways Administration**

![Image of the M365 learning pathways admin page](media/sss-sss-admin.png)

> 7.	Select the … (ellipse), then select **Add to Content Pack**

Reviewer: ```suggestion
7.	Select the **…** (ellipse), then select **Add to Content Pack**
```
